doc/templates/build-wg-meeting-0000-00-00.md

```diff
@@ -0,0 +1,33 @@
+# Node.js Foundation Build WG Meeting YYYY-MM-DD
+
+[GitHub issue]()
+[Meeting video]()
+[Previous meeting]()
+
+Next meeting: DD MMM YYYY
+
+## Present
+- Michael Dawson (@mhdawson)
+- Johan Bergström (@jbergstroem)
+- João Reis (@joaocgreis)
+- Gibson Fahnestock (@gibfahn)
+- Michele Capra (@piccoloaiutante)
+- Kyle Farnung (@kfarnung)
```

Comment:
How about leaving just one name as place holder as the group will change from one standup to another?